src/test/java/mil/dds/anet/test/resources/PersonResourceTest.java

```diff
@@ -450,6 +484,46 @@ public void mergePeopleTest() {
         POSITION_FIELDS + " person {" + PERSON_FIELDS + " }", created.getUuid(),
         new TypeReference<GraphQlResponse<Position>>() {});
     assertThat(winnerPos.getPerson()).isEqualTo(winner);
+
+    // Re-create loser and put into the position.
+    loser = new Person();
+    loser.setRole(Role.ADVISOR);
+    loser.setName("Loser for Merging");
+    loserUuid = graphQLHelper.createObject(admin, "createPerson", "person", "PersonInput", loser,
+        new TypeReference<GraphQlResponse<Person>>() {});
+    assertThat(loserUuid).isNotNull();
+    loser = graphQLHelper.getObjectById(admin, "person", FIELDS, loserUuid,
+        new TypeReference<GraphQlResponse<Person>>() {});
+    variables = new HashMap<>();
+    variables.put("winnerUuid", winnerUuid);
+    variables.put("loserUuid", loserUuid);
+    Instant now = Instant.now();
```

Comment:
Can be `final` (note that Checkstyle complains about this as well: _Distance between variable 'now' declaration and its first usage is 7, but allowed 3.  Consider making that variable final if you still need to store its value in advance (before method calls that might have side effects on the original value). [VariableDeclarationUsageDistance]_ ).